projects/Mallard/src/components/front/cards.tsx

```diff
@@ -18,24 +20,28 @@ const styles = StyleSheet.create({
         flexBasis: '100%',
     },
 })
+interface SmallCardProps {
+    style: StyleProp<ViewStyle>
+    article: FrontArticle
+}
+type InjectedProps = NavigationInjectedProps<Params>
```

Comment:
this is not the correct typing as `navigation.navigate(route, params)` can call any route as the first parameter. With this typing, `navigation.navigate('Home')` or `navigation.navigate('Settings')` would be type checked as illegal even though they are correct

Reply:
This typing is on `params` not `route`. I'm still trying to figure out whether it's possible to type specific `route` and `parameter` combinations.

Reply:
The type checking seems to work best with `.getParam`

Reply:
I went with simple examples but `navigation.navigate('Settings')` would be missing a second parameter and something like `navigation.navigate('issue',{id: 12})` wouldn't work for sure. I appreciate the underlying idea but I think having a looks-right-but-not-quite type here makes the code quite confusing to grok.

Again, as discussed, I think typing the navigation should be its own separate task as it carries a lot of complexity to get right. I don't understand why it's part of this PR

Reply:
I don't think removing this until we come back to type navigation properly is worth it. Can we add another card for this work?